opentelemetry-sdk/src/logs/log_processor.rs

```diff
@@ -190,7 +190,9 @@ impl<T: LogExporter> LogProcessor for SimpleLogProcessor<T> {
         self.is_shutdown
             .store(true, std::sync::atomic::Ordering::Relaxed);
         if let Ok(mut exporter) = self.exporter.lock() {
-            exporter.shutdown();
+            exporter
+                .shutdown()
+                .map_err(|e| LogError::Other(Box::new(e)))?;
```

Comment:
this should be ShutdownError::InternalFailure? 
Same with the else branch here dealing with mutexpoison.

Reply:
@cijothomas to be clear you're suggesting that we change `trait LogProcessor`'s `shutdown` signature as well, right?

Reply:
yes